paddle_fl/core/trainer/fl_trainer.py

```diff
@@ -93,22 +97,72 @@ def set_trainer_job(self, job):
     def reset(self):
         self.cur_step = 0
 
-    def run(self, feed, fetch):
+    def run(self, feed, fetch, train_id, mask):
         self._logger.debug("begin to run FedAvgTrainer, cur_step=%d, inner_step=%d" %
                            (self.cur_step, self._step))
         if self.cur_step % self._step == 0:
             self._logger.debug("begin to run recv program")
             self.exe.run(self._recv_program)
+        scope = fluid.global_scope()
+        print("****** fc_0.b_0: {0}".format(numpy.array(scope.find_var("fc_0.b_0").get_tensor())))
+        print("****** fc_0.w_0: {0}".format(numpy.array(scope.find_var("fc_0.w_0").get_tensor())))
+        self._logger.debug("begin to run current step")
+        loss = self.exe.run(self._main_program, 
+                     feed=feed,
+                     fetch_list=fetch)
+        if self.cur_step % self._step == 0:
+            self._logger.debug("begin to run send program")
+            scope = fluid.global_scope()
+            name1 = "fc_0.b_0.opti.trainer_" + str(train_id)
```

Comment:
It seems that this is hard coding which is not a general case for other models.

Reply:
Thanks for pointing out this bug. This code is for debug. Now I have removed them.